pkg/asset/installconfig/gcp/session.go

```diff
@@ -78,40 +79,70 @@ func getCredentials(ctx context.Context) (*googleoauth.Credentials, error) {
 	return creds, nil
 }
 
+func lazyInit() {
```

Comment:
not a fan of the function name.

Reply:
Changed to `setup()`.

Reply:
Nevermind, I'm removing the function and making that logic part of `loadCredentials`.